src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
Security Concern: The database URL is hardcoded in the source code. Consider externalizing this to a configuration file or environment variable to enhance security.